platform/linux-generic/odp_crypto_openssl.c

```diff
@@ -37,7 +37,7 @@
 #define _ODP_HAVE_CHACHA20_POLY1305 0
 #endif
 
-#define MAX_SESSIONS 32
+#define MAX_SESSIONS 4000
```

Comment:
How much memory does this take?

Reply:
On my x86_64 system with OpenSSL 1.1.1f, roughly this much more, if I looked right:
- 2.3 MB for the process
- 0.15 MB for each thread of the process
- 3,3 MB + some amount of malloc overhead for each ODP thread (unless crypto has been disabled in the runtime config)

This is a bit unrelated but the usage of malloc inside the openssl library is actually a bit problematic since it won't work in the process mode where each ODP thread is an OS process. Suitable hooking could fix that with some effort, though.